e2e/tests/oai/002.sh

```diff
@@ -27,18 +27,26 @@ source "${LIBDIR}/k8s.sh"
 # shellcheck source=e2e/lib/kpt.sh
 source "${LIBDIR}/kpt.sh"
 
+#adding directory path to register repository to have upstream packages in repository subdirectory when used with packagevariantset
+#TODO: to change the private catalog repository to nephio catalog repository after PR approval
+kpt alpha repo register --namespace default https://github.com/josephthaliath/catalog.git --directory=workloads/oai --name=oai-ran-packages
```

Comment:
@josephthaliath are you using this [PR](https://github.com/nephio-project/catalog/pull/11) for these tests?